src/legacy/core_plugins/interpreter/index.ts

```diff
@@ -28,7 +28,16 @@ export default function InterpreterPlugin(kibana: any) {
     require: ['kibana', 'elasticsearch'],
     publicDir: resolve(__dirname, 'public'),
     uiExports: {
-      injectDefaultVars: server => ({ serverBasePath: server.config().get('server.basePath') }),
+      injectDefaultVars: server => ({
+        serverBasePath: server.config().get('server.basePath'),
+        interpreterConfig: server.config().get('interpreter'),
+      }),
+    },
+    config: (Joi: any) => {
+      return Joi.object({
+        enabled: Joi.boolean().default(true),
```

Comment:
Is the `enabled` config currently needed, or just `enableInVisualize`?

Reply:
just enableInVisualize, without enabled thing doens't work however. I assume its the default option of every plugin ? not really sure why that's the case, but i followed same patterns used in vega.